generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -1601,126 +1630,126 @@ def get_cal_info_attribute_bool(self, device_name, attribute):
         response = self._invoke(
             self._client.GetCalInfoAttributeBool,
             grpc_types.GetCalInfoAttributeBoolRequest(
-            attribute=attribute, device_name=device_name))
+                attribute=attribute, device_name=device_name))
         return response.value
 
     def get_cal_info_attribute_double(self, device_name, attribute):
         response = self._invoke(
             self._client.GetCalInfoAttributeDouble,
             grpc_types.GetCalInfoAttributeDoubleRequest(
-            attribute=attribute, device_name=device_name))
+                attribute=attribute, device_name=device_name))
         return response.value
 
     def get_cal_info_attribute_string(self, device_name, attribute):
         response = self._invoke(
             self._client.GetCalInfoAttributeString,
             grpc_types.GetCalInfoAttributeStringRequest(
-            attribute=attribute, device_name=device_name))
+                attribute=attribute, device_name=device_name))
         return response.value
 
     def get_cal_info_attribute_uint32(self, device_name, attribute):
         response = self._invoke(
             self._client.GetCalInfoAttributeUInt32,
             grpc_types.GetCalInfoAttributeUInt32Request(
-            attribute=attribute, device_name=device_name))
+                attribute=attribute, device_name=device_name))
         return response.value
 
     def get_chan_attribute_bool(self, task, channel, attribute):
         response = self._invoke(
             self._client.GetChanAttributeBool,
             grpc_types.GetChanAttributeBoolRequest(
-            attribute=attribute, channel=channel, task=task))
+                attribute=attribute, channel=channel, task=task))
         return response.value
 
     def get_chan_attribute_double(self, task, channel, attribute):
         response = self._invoke(
             self._client.GetChanAttributeDouble,
             grpc_types.GetChanAttributeDoubleRequest(
-            attribute=attribute, channel=channel, task=task))
+                attribute=attribute, channel=channel, task=task))
         return response.value
 
     def get_chan_attribute_double_array(self, task, channel, attribute):
         response = self._invoke(
             self._client.GetChanAttributeDoubleArray,
             grpc_types.GetChanAttributeDoubleArrayRequest(
-            attribute=attribute, channel=channel, task=task))
+                attribute=attribute, channel=channel, task=task))
         return response.value
 
     def get_chan_attribute_int32(self, task, channel, attribute):
         response = self._invoke(
             self._client.GetChanAttributeInt32,
             grpc_types.GetChanAttributeInt32Request(
-            attribute=attribute, channel=channel, task=task))
+                attribute=attribute, channel=channel, task=task))
         return response.value
```

Comment:
This also affects arrays of enums.
https://github.com/ni/nidaqmx-python/blob/master/src/codegen/protos/nidaqmx.proto#L7148

```
message GetDeviceAttributeInt32ArrayResponse {
  int32 status = 1;
  repeated DeviceInt32AttributeValues value = 2;
  repeated int32 value_raw = 3;
}
```

Reply:
Thanks for pointing this out Brad. Major reason behind this behavior is that the `enum` tag is missing for these parameters in the functions metadata. So the metadata has to be updated according and hopefully with minor tweaks we should be able to fix this. This will be taken care of in [Task 2375627](https://dev.azure.com/ni/DevCentral/_workitems/edit/2375627): Update the metadata with the missing enum values and generate the gRPC interpreter